lab-padget/server.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const http = require('http');
+const Router = require('./lib/router');
+const storage = require('./lib/storage');
+const LiveShow = require('./model/live-show'); // was >> KidToy ./model/kid-toys
+const debug = require('debug')('http:server');
+const PORT = process.env.PORT || 3000;
+
+const router = new Router();
+const server = module.exports = http.createServer(router.route());
+// Yesterday we wrote: (function(req, res)). See cowsay and see the callback.
```

Comment:
TA JR says: remove comments. enterprise code shouldn't have comments, nor should it have console.logs.
JR says: do what makes sense and helps you figure out the flow of your app, then make a PR with clean code.